docs/source/index.rst

```diff
@@ -3,7 +3,7 @@ Netengine
 =========
 
 **Netengine** is a python library that aims to provide a single API to extract common
-information from network devices using different protocols (eg: SNMP, SSH, HTTP) and different firwmares (eg: OpenWRT, AirOS).
+information from network devices using the Simple Network Management Protocol (SNMP) and different firwmares (eg: OpenWRT, AirOS).
```

Comment:
it would be nice to mention that implementing backends that retrieve info in other ways is possible if needed.

Reply:
Done, please take a look